### PR TITLE
Implement basic comment translation

### PR DESCRIPTION
**Pull request by shulijones**
Implement basic comment translation into English and Spanish.

Features to come:
-Add additional languages
-Translate the date as well as the comment's message

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h2>Shuli's Portfolio</h2>
     </div> 
     
     <div id="guest-book">
-      <form action="/guestbook?maxcomments=5" method="POST">
+      <form action="/guestbook?maxcomments=5&lang=en" method="POST">
```
Comment (kyracq) [inline portfolio/src/main/webapp/index.html:38]: In your other files maxcomments has a dash in it which is missing here

**Review by kyracq — APPROVED** [review]
LGTM!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (shulijones) [reply portfolio/src/main/webapp/index.html:38]: Nice catch!

**Review by anicr7 — APPROVED** [review]

**Review by otobin — APPROVED** [review]
LGTM!

**Pull request merged**